pallets/aleph/src/lib.rs

```diff
@@ -121,6 +127,29 @@ pub mod pallet {
         pub(crate) fn set_next_emergency_finalizer(emergency_finalizer: T::AuthorityId) {
             <NextEmergencyFinalizer<T>>::put(emergency_finalizer);
         }
+
+        pub(crate) fn set_next_aleph_bft_version_change(version_change: VersionChange) {
+            let previous_version_change = <AlephBFTVersionChange<T>>::get();
+            let previous_session = previous_version_change.session;
+            let previous_version = previous_version_change.version_incoming;
+
+            let session = version_change.session;
+            let version = version_change.version_incoming;
+
+            assert!(
+                session > previous_session,
+                "Tried to schedule an AlephBFT version change for a session ({}) prior to the session ({}) of the currently scheduled version change.",
+                session,
+                previous_session
+            );
```

Comment:
this should be permitted imho

Reply:
Correct, will change it to a `current_session` check.